db/db_impl/db_impl.h

```diff
@@ -1131,7 +1131,13 @@ class DBImpl : public DB {
       }
     }
   };
-
+  struct LogContext {
```

Comment:
Add newlines around this struct definition.

Reply:
done